packages/commonwealth/test/unit/unitHelpers.ts

```diff
@@ -0,0 +1,11 @@
+import { TypedRequestQuery, TypedResponse } from 'server/types';
```

Comment:
Are these used anywhere?

Reply:
There is a second issue with test speed. Basically there seem to be 2 ways to perform these tests.
1. load the whole app.ts, which gets all the endpoints. The problem with this way is it takes extremely long to load
2. call the methods that process the request by itself (ex, calling getComments instead of GET api/comments)

Since 1 takes too long to load due to it needing to basically build the whole project without webpack, I had created tested these endpoints using method 2. For this reason we need to use these helper methods in this class to build the express Request and Response to pass into the methods that process the request (ex `getComments(models:DB, req: TypedRequestQuery<GetCommentsReq>,res: TypedResponse<GetCommentsResp>)`.

Although thinking about it now, maybe testing it this way is not good, since there could be some issues that testing this way might not catch, what do you think?

Reply:
I would call (1) an integration test and (2) a unit test. Both are good but unit tests are better.